src/ekore/anomalous_dimensions/unpolarized/time_like/as2.py

```diff
@@ -0,0 +1,687 @@
+"""The unpolarized time-like NLO Altarelli-Parisi splitting kernels."""
+
+import numba as nb
+import numpy as np
+from numpy import power as npp
+
+from eko import constants
+
+from ....harmonics import w1, w2, w3
+from ....harmonics.constants import zeta2, zeta3
+from ....harmonics.polygamma import cern_polygamma as polygamma
+
+
+@nb.njit(cache=True)
+def gamma_nsp(N, nf):
+    r"""Compute the NLO non-singlet positive anomalous dimension.
+
+    Implements :eqref:`B.7` from :cite:`Mitov:2006wy`.
+
+    Parameters
+    ----------
+    N : complex
+        Mellin moment
+    nf : int
+        No. of active flavors
+
+    Returns
+    -------
+    gamma_nsp : complex
+        NLO non-singlet positive anomalous dimension
+        :math:`\gamma_{ns}^{(1)+}(N)`
+
+    """
+    NS = N * N
+    NT = NS * N
+    NFO = NT * N
+
+    N1 = N + 1
+    N2 = N + 2
+    N1S = N1 * N1
+    N1T = N1S * N1
+
+    S1 = w1.S1(N)
+    S2 = w2.S2(N)
+
+    N3 = N + 3
+    N4 = N + 4
+    N5 = N + 5
+    N6 = N + 6
+
+    S11 = S1 + 1 / N1
+    S12 = S11 + 1 / N2
+    S13 = S12 + 1 / N3
+    S14 = S13 + 1 / N4
+    S15 = S14 + 1 / N5
+    S16 = S15 + 1 / N6
+
+    ZETA2 = zeta2
+    ZETA3 = zeta3
+
+    SPMOM = (
```

Comment:
you can do here or later (i.e. with #202 ): remember that we use different implementations - this should be something like $g_3$, if I remember correctly

Reply:
let's leave this for the time being and come back during #202 . As these are taken from Vogt's parametrized implementation, it's safer to let them be for the time being (as computationally they will only make a difference after #202).